### DISCUSSION TITLE
Scope of the project

### DISCUSSION THREAD

**Discussion by fordacious** [google/model-viewer#2967] [General]
Hey @elalish, I'm just curious what the scope of the model viewer project is. e.g. is there intent to implement interaction events like click, or 3d based selection events etc...

**Comment by elalish** [comment]
The scope changes slowly based on user requests; mostly we do not plan  to expose all of three.js; this is intended to be a simpler API to use at a higher level. However, we do have some functions that help with click interactivity, so long as you separate your model into different materials: https://modelviewer.dev/docs/#entrydocs-scenegraph-methods-materialFromPoint
